# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ncurses/dont_remove_library-premake.diff
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./ncurses/Makefile.orig	2005-01-19 18:44:35.000000000 -0300
+++ ./ncurses/Makefile	2005-01-19 18:46:32.000000000 -0300
@@ -617,9 +617,8 @@
 install.libs \
 install.ncurses :: $(DESTDIR)$(libdir) $(LIBRARIES)
 	@echo installing ../lib/libncurses.so.$(REL_VERSION) as $(DESTDIR)$(libdir)/libncurses.so.$(REL_VERSION)
-	-@rm -f $(DESTDIR)$(libdir)/libncurses.so.$(REL_VERSION)
 	$(INSTALL_LIB) ../lib/libncurses.so.$(REL_VERSION) $(DESTDIR)$(libdir)/libncurses.so.$(REL_VERSION)
-	cd $(DESTDIR)$(libdir) && (rm -f libncurses.so.$(ABI_VERSION); $(LN_S) libncurses.so.$(REL_VERSION) libncurses.so.$(ABI_VERSION); rm -f libncurses.so; $(LN_S) libncurses.so.$(ABI_VERSION) libncurses.so; )
+	cd $(DESTDIR)$(libdir) && ($(LN_S) -f libncurses.so.$(REL_VERSION) libncurses.so.$(ABI_VERSION); $(LN_S) -f libncurses.so.$(ABI_VERSION) libncurses.so; )
 	- test -z "$(DESTDIR)" && /sbin/ldconfig
 
 uninstall \
